frontend/src/components/project/overview/Access.tsx

```diff
@@ -0,0 +1,266 @@
+import {
+  Dialog,
+  Icon,
+  InputWrapper,
+  List,
+  Radio,
+  Tooltip,
+} from "@equinor/eds-core-react";
+import { createFormHook } from "@tanstack/react-form";
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { useState } from "react";
+import { toast } from "react-toastify";
+
+import { FmuProject } from "#client";
+import {
+  projectGetProjectQueryKey,
+  projectPatchAccessMutation,
+} from "#client/@tanstack/react-query.gen";
+import { Access } from "#client/types.gen";
+import { Classification } from "#client/types.gen";
+import {
+  CancelButton,
+  GeneralButton,
+  SubmitButton,
+} from "#components/form/button";
+import { TextField } from "#components/form/field";
+import {
+  EditDialog,
+  InfoBox,
+  PageCode,
+  PageHeader,
+  PageList,
+  PageSectionSpacer,
+  PageText,
+} from "#styles/common";
+import { fieldContext, formContext } from "#utils/form";
+import { requiredStringValidator } from "#utils/validator";
+
+const { useAppForm: useAppFormAccessEditor } = createFormHook({
+  fieldComponents: {
+    TextField,
+    Radio,
+  },
+  formComponents: {
+    SubmitButton,
+    CancelButton,
+  },
+  fieldContext,
+  formContext,
+});
+
+function AccessEditorForm({
+  accessData,
+  isDialogOpen,
+  setIsDialogOpen,
+}: {
+  accessData: Access | null | undefined;
+  isDialogOpen: boolean;
+  setIsDialogOpen: (open: boolean) => void;
+}) {
+  const closeDialog = ({ formReset }: { formReset: () => void }) => {
+    formReset();
+    setIsDialogOpen(false);
+  };
+  const queryClient = useQueryClient();
+  const { mutate, isPending } = useMutation({
+    ...projectPatchAccessMutation(),
+    onSuccess: () => {
+      void queryClient.invalidateQueries({
+        queryKey: projectGetProjectQueryKey(),
+      });
+    },
+  });
+
+  const form = useAppFormAccessEditor({
+    defaultValues: {
+      assetName: accessData?.asset.name ?? "",
+      classification: accessData?.classification ?? "",
+    },
+
+    onSubmit: ({ value, formApi }) => {
+      mutate(
+        {
+          body: {
+            asset: { name: value.assetName.trim() },
+            classification: value.classification as Classification,
+          },
+        },
+        {
+          onSuccess: () => {
+            toast.info(`Successfully set access information`);
```

Comment:
Regular quotation marks should be used instead of backticks, which are used for template strings.
```suggestion
            toast.info("Successfully set access information");
```